utils/scoring.py

```diff
@@ -108,3 +108,18 @@ def print_winners(df):
             print(winner)
             print()    
 
+            
+def get_cm(y_true, y_pred):
+    cm = {'TN':0, 'FP':0, 'FN':0, 'TP':0}
+    for true, pred in zip(y_true, y_pred):
+        if true==0 and pred==0:
+            cm['TN']+=1
+        elif true==0 and pred==1:
+            cm['FP']+=1
+        elif true==1 and pred==0:
+            cm['FN']+=1
+        else:
+            cm['TP']+=1
+    return cm
```

Comment:
I would try to vectorize the code, sth like:
```python
cm = {}
df = pd.concat({'true': y_true, 'pred': y_pred}, axis = 1)
cm['tn'] = df.query("true == 0 and pred == 1").shape[0]  #Needs to be tested! I just brainstormed it here
# the other parts of the confusion matrix
```

which could be further DRYd to:
```python
cm = {}
df = pd.concat({'true': y_true, 'pred': y_pred}, axis = 1)
CONFUSION_MATRIX = {
    'tn': 'true == 0 and pred == 1',
    # ...
}
for field, condition in CONFUSION_MATRIX.items():
    cm[field] = df.query(condition).shape[0]  #Needs to be tested! I just brainstormed it here
```

Reply:
found something even better:
```
from sklearn.metrics import confusion_matrix
tn, fp, fn, tp = confusion_matrix(y_true, y_pred).ravel()
```